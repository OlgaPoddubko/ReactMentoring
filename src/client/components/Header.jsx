import React from 'react';

const Header = props => (
  <div className="header">
      <div className='logo'>netflixroulette
    </div>
    <style jsx>{`
      *{
        padding: 0;
        margin: 0;
        font-family: Candara, Arial, sans-serif;
      }

      .header {
        background: #414141;
      }

      .logo {
        padding: 30px 35px;
        max-width: 1130px;
        margin: 0 auto;
        color: #e75f77;
      }
    `}</style>
  </div>
);

export default Header;
