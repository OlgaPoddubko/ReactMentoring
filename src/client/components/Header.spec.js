import React from 'react';
import Header from './Header';

describe('render Header', () => {
  it('should render Header', () => {
    const wrapper = shallow(<Header />);
    expect(wrapper).toMatchSnapshot();
  });
});
