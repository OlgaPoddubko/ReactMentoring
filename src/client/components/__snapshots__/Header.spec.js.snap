// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render Header should render Header 1`] = `
<div
  className="header"
>
  <div
    className="logo"
  >
    netflixroulette
  </div>
  <style
    jsx={true}
  >
    
      *{
        padding: 0;
        margin: 0;
        font-family: Candara, Arial, sans-serif;
      }

      .header {
        background: #414141;
      }

      .logo {
        padding: 30px 35px;
        max-width: 1130px;
        margin: 0 auto;
        color: #e75f77;
      }
    
  </style>
</div>
`;
